src/v2/Components/Menu.tsx

```diff
@@ -107,9 +109,11 @@ const MenuLink = styled(RouterLink)<{ hasLighterTextColor: boolean }>`
 
   ${display};
 
-  &:hover {
-    background-color: ${p =>
-      p.hasLighterTextColor ? color("black10") : color("black5")};
+  &:hover,
+  &:focus {
+    outline: none;
```

Comment:
Same styles for focus, removing the outline that would be added through focus-visible.